explorer/explorer.go

```diff
@@ -41,18 +94,71 @@ func ListenEvents(event event.Event) {
 //
 // In some platforms the resulting `io.ReadCloser` is a `os.File`, but it's not
 // a guarantee.
-func ReadFile(extensions ...string) (io.ReadCloser, error) {
-	return openFile(extensions...)
+//
+// It's a blocking call, you should call it on a separated goroutine. For most OSes, only one
+// Import or Export, can happen at the same time, for each app.Window/Explorer.
+func (e *Explorer) Import(extensions ...string) (io.ReadCloser, error) {
```

Comment:
What motivated the name change from `ReadFile` to `Import`?

Reply:
The reason is because it doesn't match with `os.ReadFile()` anymore. I choose `ReadFile` and `WriteFile` when the API returns `[]byte`, which I change to `io.ReadCloser` later. _Also, internally, there's many functions using `Open`, `Write`, `Read` and it was confusing, since it also overlaps with the FileReader/FileWriter (which also have Write/Read functions)._ I steal that name from macOS/iOS API. :P 

That was the reason, but I can switch back to ReadFile.

Reply:
I agree that using `ReadFile` and `WriteFile` would be confusing given the mismatch in their signatures. What would you think of `OpenFileWithExtensions` and `CreateFile`? Conceptually, these operations are close to the `os.Open` family of functions. You get back a type implementing io interfaces, and you manipulate that type with read or write.

Reply:
I don't know about `Open`/`OpenFile`. Because on `os` it returns `os.File`, then it's possible to read and write at same-time. 

Currently, it's not possible to select a file (Open) to edit the content. _That only works on Windows/macOS, using cast to `os.File`._

Also, I'm not sure what would be the names to "select folder" (explorer.Folder()?) and "select multiples files" (ImportMultiple? OpenFile**s**?).

Reply:
I think that the function signature only returning an `io.ReadCloser` makes it fairly clear that the file can only reliably be read. However, what would you think of `ChooseFileWithExtensions`? `ChooseDirectory`? `ChooseFiles`? I think that's clear and doesn't imply that the files are writable.